### PR TITLE
ok swad

### PR DESCRIPTION
**Pull request by marcogravbrot**
i know you're late but i know you can do it! i believe in you!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -329,8 +329,13 @@ <h1 style="font-size: 2em; font-weight: lighter;">rolling in 1 line cac bypasses
...
             "velkon"
         );
+        
+        glua.team.addMember(
+            "76561198076402038",
+            "https://steamcdn-a.akamaihd.net/steamcommunity/public/images/avatars/dd/dd8353bfc0df7be500c5fdcebf4e544fe155d2dc_full.jpg",
+            "author."
+        );
 
-        glua.team.addFriend("76561198076402038", "Author.");
```
Comment (FredrickSmith) [inline index.html:333]: incorrect usage of the delete key

**Comment by WansonSwanson** [comment]
This needs to be accepted

**Comment by SwadicalRag** [comment]
test comment please ignore

**Comment by marcogravbrot** [comment]
ok ignored

**Pull request closed without merging**